### PR TITLE
Christina sort

### PR DESCRIPTION
**Pull request by ChristinaGislason**

### CODE REVIEW

File: sort/challenges-12.test.js [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+// Write a function named sortBackwards that takes in an array of numbers and returns the same array,
+// with the numbers sorted, highest to smallest.
+// ------------------------------------------------------------------------------------------------
+
+const sortBackwards = (nums) => {
+  nums.sort(function(a,b) {
+    return b - a;
+  });
+  return nums;
+};
```
Comment (zarkle) [inline sort/challenges-12.test.js:15]: you can combine these into one line `return nums.sort( (a,b) => b - a );`

File: sort/challenges-12.test.js [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+// with the strings sorted by their length, lowest to highest.
+// ------------------------------------------------------------------------------------------------
+
+const sortByLength = (strings) => {
+ 
+  strings.sort(function(a , b) {
+    return a.length - b.length;
+  });
+  return strings;
+};
```
Comment (zarkle) [inline sort/challenges-12.test.js:43]: same as challenge 1, one line `return strings.sort( (a,b) => a.length - b.length );`

Comment (ChristinaGislason) [reply sort/challenges-12.test.js:43]: updated!

File: sort/challenges-12.test.js (hunk 1 above)
Comment (ChristinaGislason) [reply sort/challenges-12.test.js:15]: updated it!

**Pull request merged**